client/karma.js

```diff
@@ -175,21 +147,8 @@ var Karma = function (socket, iframe, opener, navigator, location) {
     }
   }
 
-  var UNIMPLEMENTED_START = function () {
-    this.error('You need to include some adapter that implements __karma__.start method!')
-  }
-
-  // all files loaded, let's start the execution
-  this.loaded = function () {
-    // has error -> cancel
-    if (!hasError) {
-      this.start(this.config)
-    }
-
-    // remove reference to child iframe
-    this.start = UNIMPLEMENTED_START
-  }
-
+  // TODO: Does anyone use `this.store`???
+  //   https://gitter.im/karma-runner/karma?at=56e48e07618c335373eb497f
```

Comment:
I'm pretty sure it's no longer used. It was introduced in this commit which is 4 years old and apparently when the Jasmine adapter was still in this repo o_o

https://github.com/karma-runner/karma/commit/eedcbbd9cdc5d13b09bcef7dd12b1462a2a7a155

I can't find any references to `store` in `karma-jasmine` or any other spot checked repos so I think we are good to remove it:

https://github.com/karma-runner/karma-jasmine/search?utf8=%E2%9C%93&q=store